modules/workflow_config/workflow_config/inputs.tf

```diff
@@ -0,0 +1,32 @@
+variable "namespace_annotations" {
+  type        = map(string)
+  description = "Map of annotations applied to the created namespace"
+  default     = {}
+}
+
+variable "namespace" {
+  description = "Name of namespace"
+  type        = string
+  default     = "swoop"
+}
+
+variable "aws_access_key" {
+  description = "AWS Access Key ID"
+  type        = string
+  default     = ""
+  sensitive   = true
+}
+
+variable "aws_secret_access_key" {
+  description = "AWS Secret Access Key"
+  type        = string
+  default     = ""
+  sensitive   = true
+}
+
+variable "aws_region" {
+  description = "AWS Region"
+  type        = string
+  default     = ""
+  sensitive   = true
+}
```

Comment:
I think we're missing the optional session token that was recently added to the helm chart